x-pack/packages/security-solution/ecs_data_quality_dashboard/impl/data_quality/data_quality_panel/tabs/incompatible_tab/helpers.ts

```diff
@@ -65,11 +66,37 @@ export const getIncompatibleMappings = (
 ): EnrichedFieldMetadata[] =>
   enrichedFieldMetadata.filter((x) => !x.isEcsCompliant && x.type !== x.indexFieldType);
 
+export const getIncompatibleMappingsFields = (
+  enrichedFieldMetadata: EnrichedFieldMetadata[]
+): string[] =>
+  enrichedFieldMetadata.reduce((acc, x) => {
+    if (!x.isEcsCompliant && x.type !== x.indexFieldType && x.indexFieldName != null) {
+      const field = escape(x.indexFieldName);
+      if (field != null) {
+        acc.push(field);
```

Comment:
consider replacing `push` with `...` for consistency with the the existing code here and in the `getIncompatibleValuesFields` function below